xml/Microsoft.Win32/SaveFileDialog.xml

```diff
@@ -124,7 +124,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets a value indicating whether the dialog box will attempt to create a test file at the selected path (default is <see langword="true"/>). If this flag is not set, the calling application must handle errors, such as denial of access, discovered when the item is created.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value><see langword="true"/> if the dialog box will attempt to create a test file at the selected path; otherwise, <see langword="false"/>. The default is <see langword="true"/>.</value>
```